packages/xo-web/src/xo-app/hub/index.js

```diff
@@ -0,0 +1,60 @@
+import _ from 'intl'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import React from 'react'
+import { addSubscriptions } from 'utils'
+import { Container, Col, Row } from 'grid'
+import { injectState, provideState } from 'reaclette'
+import { isEmpty, map, orderBy } from 'lodash'
+import { subscribeResourceCatalog } from 'xo'
+
+import Page from '../page'
+import Resource from './resource'
+
+// ==================================================================
+
+const HEADER = (
+  <h2>
+    <Icon icon='menu-hub' /> {_('hubPage')}
+  </h2>
+)
+
+export default decorate([
+  addSubscriptions({
+    catalog: subscribeResourceCatalog({ filters: { hub: true } }),
+  }),
+  provideState({
+    computed: {
+      resources: ({ availableResources }) =>
+        orderBy(availableResources, res => res.name, 'asc'),
+      availableResources: (_, { catalog }) =>
+        map(catalog, (entry, namespace) => ({
+          namespace,
+          ...entry.xva,
+        })),
+    },
+  }),
+  injectState,
+  ({ state: { resources } }) => (
+    <Page header={HEADER} title='hubPage' formatTitle>
```

Comment:
Add the [`adminOnly`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/utils.js#L628-L633) decorator so that non-admin users can't access this page.

Reply:
Decorator added on the the top of the tree